content/en/blog/2024/gateway-mesh-ga/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Gateway API Mesh Support Promoted To Stable"
```

Comment:
Thing to take care of: we're announcing, on the Istio blog, that GW API mesh use cases are stable in GW API.  They remain Beta in Istio if feature stages has that right.